tests/src/test/java/WorldTests.java

```diff
@@ -0,0 +1,179 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.anuke.arc.ApplicationCore;
+import io.anuke.arc.backends.headless.HeadlessApplication;
+import io.anuke.arc.backends.headless.HeadlessApplicationConfiguration;
+import io.anuke.arc.util.Log;
+import io.anuke.arc.util.Time;
+import io.anuke.mindustry.Vars;
+import io.anuke.mindustry.content.Blocks;
+import io.anuke.mindustry.core.GameState.State;
+import io.anuke.mindustry.core.Logic;
+import io.anuke.mindustry.core.NetServer;
+import io.anuke.mindustry.core.World;
+import io.anuke.mindustry.game.Content;
+import io.anuke.mindustry.io.BundleLoader;
+import io.anuke.mindustry.world.Tile;
+
+import static io.anuke.mindustry.Vars.content;
+import static io.anuke.mindustry.Vars.headless;
+import static io.anuke.mindustry.Vars.logic;
+import static io.anuke.mindustry.Vars.netServer;
+import static io.anuke.mindustry.Vars.state;
+import static io.anuke.mindustry.Vars.world;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class WorldTests {
+    static Tile[][] tiles;
+
+    @BeforeAll
+    static void launchApplication(){
```

Comment:
Changed WorldTests.java and performed the changes as requested. The test suite now directly invokes the method found in ApplicationTests rather than having code duplication.

Reply:
Me and my team would be highly appreciated if you could kindly have a look at these changes @Anuken.